github_connector/wizards/wizard_load_github_model.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+from ..models.github import _GITHUB_TYPE
+
+
+class WizardLoadGithubModel(models.TransientModel):
+    _name = 'wizard.load.github.model'
```

Comment:
I get this warning:
`wizard.load.github.model has no _description `
so I think It's better that you add the parameter
`_description`